### DISCUSSION TITLE
Support for the Rank Math plugin

### DISCUSSION THREAD

**Discussion by stevejburge** [publishpress/publishpress-checklists#434] [Feature Requests]
https://wordpress.org/plugins/seo-by-rank-math/

This checklist item would only appear if Rank Math is installed.

You can choose the minimum % score.

**Comment by stevejburge** [comment]
One more vote for this

**Comment by rizaardiyanto1412** [comment]
One more vote for this: https://wordpress.org/support/topic/integrate-with-rankmath-seo/

**Comment by dorzic** [comment]
One more vote for this

**Comment by CodingSamurai** [comment]
Another vote for this.

**Comment by stevejburge** [comment]
Solved with 2.18
